### ISSUE TITLE
Some questions about the code.

### ISSUE THREAD

**Issue by WHQ1111** [hytseng0509/CrossDomainFewShot#17]
As we can see,  the line 280 and the line 283 of the file 'methods/backbone.py' mean that we use the Feature Wise Transformation module in MAML not the metric-based models. But this contradicts the paper, doesn't it?

**Comment by hytseng0509** [comment]
Sorry for the confusion. The naming "maml" here indicates that we want to enable the gradient back-propagation in the learning-to-generalize (or learning-to-learn) training process.

**Comment by WHQ1111** [comment]
> Sorry for the confusion. The naming "maml" here indicates that we want to enable the gradient back-propagation in the learning-to-generalize (or learning-to-learn) training process.

So why we use the Feature Wise Transformation module when we want to enable the gradient  back-propagation in the learning-to-generalize (or learning-to-learn) training process? Is this back-propagation process similar to calculating the gradient along the inner loop in MAML?

**Comment by hytseng0509** [comment]
Yeah you are right. The code is adopted from the MAML implementation, so does the naming. :-)
